README.md

```diff
@@ -133,6 +133,8 @@ expect(valueA: any).toMatchDiffSnapshot(valueB: any, options?: Options, testName
 - `expand: boolean` (default: `false`) – expand the diff, so the whole information is preserved
 - `colors: boolean` (default: `false`) – preserve color information from Jest diff
 - `contextLines: number` (default: 5) - number of context lines to be shown at the beginning and at the end of a snapshot
+- `omitPatchMarks: boolean` (default: `false`) - prevent line number patch marks from appearing in
```

Comment:
How about `stablePatchmarks`? I'm still not sure how to best me it. @SimenB ideas?

Reply:
`stable` is better than `omit`. Other possible verbs:

normalize
scrub

`normalize` doesn't communicate a whole lot. `scrub` communicates what we are doing to the patch mark, while `stable` communicates the purpose behind it.

Reply:
I think I'd still go with `stable`

Reply:
all set